src/js/components/utils/pageDescription.tsx

```diff
@@ -20,23 +18,13 @@ const PageDescription = ({
   const renderedDescription = (
     <>
       {!descriptionHasTitle && (
-        <div className="slds-is-relative">
-          <h2
-            className="slds-truncate slds-text-heading_small slds-m-bottom_small"
-            title={renderedTitle}
-          >
-            {renderedTitle}
-            <TourPopover
-              align="top"
-              heading={i18n.t('Metecho Project')}
-              body={i18n.t(
-                'This is the image, name, and description of the Project. Metecho Projects are equivalent to Repositories on GitHub.',
-              )}
```

Comment:
@jgerigmeyer Are we planning to remove this one for now?

Reply:
@dvdherron That's my proposal, yes. I added the "Metecho Projects are equivalent to Repositories on GitHub" text to the other popover on the project name. But this text is both self-evident and also not always true. That is, some projects don't have images, and this isn't always the name of the project -- so all we can really say here is "This is the description of the Project", which doesn't seem all that useful of a popover IMO.